app/controllers/api/v1/short_form_controller.rb

```diff
@@ -64,14 +64,16 @@ def show_listing_application_for_user
   def submit_application
     response = ShortFormService.create_or_update(application_params, applicant_attrs)
     if response.present?
-      attach_files_and_send_confirmation(response)
-      if current_user && application_complete
-        delete_draft_application(application_params[:listingID])
-      end
+      process_submit_app_response(response)
```

Comment:
this was refactored due to cyclomatic complexity alert by rubocop -- other ways to refactor the rescue instead of this is welcome :)